src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisDetails.jsx

```diff
@@ -49,6 +49,13 @@ export default function AnalysisDetails() {
         </Paragraph>
       )
     },
+    {
+      title: i18n("AnalysisDetails.description"),
+      desc:
+        analysisDetailsContext.analysisDescription !== ""
+          ? analysisDetailsContext.analysisDescription
+          : "N/A"
```

Comment:
N/A is english so you might want to internationalize that too.

Reply:
Added in ee4ce5bfecdbc07e523bc69a4be9ed492a8d051b